config/routes.rb

```diff
@@ -16,7 +16,11 @@
       get '/curriculums', to: 'api_curriculums#index'
 
       get '/curriculums/:curriculum_id', to: 'api_curriculums#show'
-      get '/current_session', to: 'api_session#index'
+
+      get '/curriculums/:curriculum_id/learning_units',
+        to: 'api_learning_units#learning_units_of_curriculum'
+
+        get '/current_session', to: 'api_session#index'
```

Comment:
this line should be one indentation level less

Reply:
done, ty